src/codelabs/null-safety.md

```diff
@@ -536,19 +536,19 @@ void main() {
 
 ## What's next?
 
-Congratulations, you've finished the codelab! If you'd like to learn more, here
+Congratulations, you've finished the tutorial! If you'd like to learn more, here
 are some suggestions for where to go next:
 
 -   Learn more about null safety:
     -   Overview: [Sound null safety](/null-safety).
     -   Deep dive:
         [Understanding null safety](/null-safety/understanding-null-safety).
--   Try another [codelab](/codelabs).
+-   Try another [tutorial](/tutorials).
 -   [Get the Dart SDK](/get-dart).
 
-If you're interested in using embedded DartPads, like this codelab does, see
+If you're interested in using embedded DartPads, like this tutorial does, see
 [best practices for using DartPad in tutorials](/tools/dartpad/dartpad-best-practices).
-If you're interested in improving this codelab, see
+If you're interested in improving this tutorial, see
 [issue #3093][].
```

Comment:
**question:** This page isn't about DartPad, so I would delete that reference. Also, I don't recommend linking to the issue. The page has a button that can be clicked to review the code, then create an issue or PR.